docs/dev/test-configuration-options.md

```diff
@@ -598,7 +598,7 @@ Enables the interception of biometric input, allowing the test to simulate Touch
 Enables the use of the app's private app container directory instead of the shared app group container directory. For testing on the Real Device Cloud, the app gets resigned, which is why the shared directory is not accessible.
 
 ---
-### `networkCapture`
+### `sauceLabsNetworkCaptureEnabled`
```

Comment:
@kristofmuhi to confirm, this one is for Appium? And `networkCapture` is for Espresso/XCUITest?

Reply:
nope. Appium is changed to networkCapture. 
so it is consistent through all platforms and frameworks!

Reply:
Thank you for clarifying! Could you or @aoleszyn pls update the PR?